src-common/main/scala/uk/gov/hmrc/play/audit/http/HttpAuditing.scala

```diff
@@ -199,30 +204,16 @@ trait HttpAuditing {
     !url.contains("/write/audit") && auditDisabledForPattern.findFirstIn(url).isEmpty
 
   protected case class HttpRequest(
-    url        : String,
     verb       : String,
+    url        : String,
+    headers    : Seq[(String, String)],
     body       : Option[HookData],
     generatedAt: Instant
   )
-
-  private def auditExtraHeaderFields(headers: Map[String, String]): Map[String, String] =
-    headers.map(t => t._1 -> Seq(t._2)).collect {
-      case ("Surrogate", v) => "surrogate" -> v.mkString(",")
-      case (k, v) if auditConnector.auditExtraHeaders ⇒ k -> v.mkString(",")
-    }
```

Comment:
It seems like the intention of this method might have been to dedupe duplicate headers by comma separating all the values. eg:

```
"ipAddress": "35.179.47.219, 127.0.0.1, 127.0.0.1"
```

I don't think this was functioning as expected. Collect isn't appropriate here but also we were passing this a map which wouldn't have duplicates.